invoicing_app/management/commands/generate_tax_receipts_new.py

```diff
@@ -168,8 +182,7 @@ def handle(self, **options):
             total_taxable_amount_with_tax_amount=Sum('mg_fee'),
             total_tax_amount=Sum('eb_tax'),
             payment_transactions_count=Count('event'),
-        ).iterator()
```

Comment:
i think that you remove the iterator accidentally here

Reply:
Fixed!